### ISSUE TITLE
ZOOAttack Sucess rate 

### ISSUE THREAD

**Issue by maithal** [Trusted-AI/adversarial-robustness-toolbox#1330]
Hi,

I am trying to implement a ZOO Attack using the ART toolbox and I am using targeted attack settings. However, the ASR is always 0%. I looked in prior issues and tried changing the model to output logits but still, the problem persists. 

attack = ZooAttack(classifier=classifier, batch_size=1, targeted=True, learning_rate=1e-3, 
                   max_iter=3000, binary_search_steps=9,initial_const=10)

Is there something I am missing? Any help will be greatly appreciated!
